itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestAcidOnTez.java

```diff
@@ -727,11 +727,11 @@ public void testBucketedAcidInsertWithRemoveUnion() throws Exception {
       LOG.warn(s);
     }
     String[][] expected2 = {
-      {"{\"writeid\":1,\"bucketid\":536936448,\"rowid\":0}\t1\t2", "warehouse/t/delta_0000001_0000001_0000/bucket_00001_0"},
-      {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":0}\t2\t4", "warehouse/t/delta_0000001_0000001_0000/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536936448,\"rowid\":1}\t5\t6", "warehouse/t/delta_0000001_0000001_0000/bucket_00001_0"},
-      {"{\"writeid\":1,\"bucketid\":536870912,\"rowid\":1}\t6\t8", "warehouse/t/delta_0000001_0000001_0000/bucket_00000_0"},
-      {"{\"writeid\":1,\"bucketid\":536936448,\"rowid\":2}\t9\t10", "warehouse/t/delta_0000001_0000001_0000/bucket_00001_0"}
+       {"{\"writeid\":2,\"bucketid\":536936448,\"rowid\":0}\t1\t2", "warehouse/t/delta_0000002_0000002_0000/bucket_00001"},
+       {"{\"writeid\":2,\"bucketid\":536870912,\"rowid\":0}\t2\t4", "warehouse/t/delta_0000002_0000002_0000/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536936448,\"rowid\":1}\t5\t6", "warehouse/t/delta_0000002_0000002_0000/bucket_00001"},
+       {"{\"writeid\":2,\"bucketid\":536870912,\"rowid\":1}\t6\t8", "warehouse/t/delta_0000002_0000002_0000/bucket_00000"},
+       {"{\"writeid\":2,\"bucketid\":536936448,\"rowid\":2}\t9\t10", "warehouse/t/delta_0000002_0000002_0000/bucket_00001"}
```

Comment:
Could you please modify the previous comment to match the code?

Reply:
Sorry, did not get understand your comment.

Reply:
In the code the previous comment contains the expected directory/file structure. We should fix that as well